jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py

```diff
@@ -43,6 +43,7 @@ def test_plugin(specviz_helper, spectrum1d):
             assert len(result_dict.get('uncertainty')) > 0
 
 
+@pytest.mark.filterwarnings('ignore', match="'W/m2/m' contains multiple slashes")
```

Comment:
I think you want something like this. See https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#pytest-mark-filterwarnings

```suggestion
@pytest.mark.filterwarnings(r'ignore:W/m2/m' contains multiple slashes')
```

Reply:
I thought I've seen both syntaxes used? Happy to switch

Reply:
Some characters are special in regex. You can play with https://regex101.com/ or maybe just generalize it to something like this:

```python
r"ignore:.* contains multiple slashes"
```

Reply:
Worked! I also forgot to catch the redshift slider warning. Let's see if it passes below (it passes on my local copy)

Reply:
Weird that this wasn't caught in the PR CI.